### ISSUE TITLE
Небольшое улучшение UI у футера до переезда на новый лендинг

### ISSUE THREAD

**Issue by KirillTheStranger** [hexlet-rus/runit#435]
### Problem description

Вся страница находится в центре, футер выравнен по левой стороне, из-за этого он выбивается из общей картины.

![image](https://github.com/hexlet-rus/runit/assets/125999183/211581b2-0771-4356-bef9-c73578af5f63)

### Proposed solution

Футер выравнен по середине, чтобы соответствовать общему дизайну страницы.

**Comment by KirillTheStranger** [comment]
Возьму в работу

**Comment by fey** [comment]
@KirillTheStranger вероятнее всего это не актуально, потому что у нас есть задача на лендос, а там футер уже другой 
https://runit.hexlet.ru/landing

**Comment by KirillTheStranger** [comment]
Да, задачку видел, я как раз и хотел сделать ее для того, чтобы пока переезд не случился - можно было еще поправить)

**Comment by fey** [comment]
Хорошо, давайте.
